CMakeLists.txt

```diff
@@ -2,7 +2,19 @@ cmake_minimum_required(VERSION 3.5)
 
 project(drogon CXX)
 
-add_library(${PROJECT_NAME} STATIC)
+option(BUILD_CTL "Build drogon_ctl" ON)
+option(BUILD_EXAMPLES "Build drogon_ctl" ON)
+option(BUILD_ORM "Build orm" ON)
+option(LIBPQ_BATCH_MODE "Use batch mode for libpq" ON)
+option(BUILD_DROGON_SHARED "Build drogon as a shared lib" ON)
+option(BUILD_TESTING "Build tests" OFF)
```

Comment:
The BUILD_TESTING is a cmake build-in flag with false value by default, so this line can be removed.